script/build.js

```diff
@@ -80,6 +80,7 @@ smith.destination(`../build/${options.buildtype}`);
 const ignoreList = [ 'memorial-benefits/*' ];
 if (options.buildtype === 'production') {
   ignoreList.push('rx/*');
+  ignoreList.push('education/apply-for-education-benefits/apply.md');
```

Comment:
There are some existing static pages under education. I think we'd exclude them if I changed it to that.